packages/celotool/src/lib/oracle.ts

```diff
@@ -429,7 +439,7 @@ export function switchToContextCluster(celoEnv: string, oracleContext: string) {
 /**
  * Switches to the AKS cluster associated with the given context
  */
-export function switchToAzureContextCluster(celoEnv: string, oracleContext: string) {
+export async function switchToAzureContextCluster(celoEnv: string, oracleContext: string) {
```

Comment:
nit: doesn't necessarily need to be `async` because there's no `await`ing going on here